lib/grandpa/message.go

```diff
@@ -56,6 +56,10 @@ type VoteMessage struct {
 	Message SignedMessage
 }
 
+func (v VoteMessage) String() string {
+	return fmt.Sprintf("round=%d, setID=%d, message=%s", v.Round, v.SetID, v.Message)
```

Comment:
nit since message is itself a struct, maybe wrap it with braces:

```suggestion
	return fmt.Sprintf("round=%d, setID=%d, message={%s}", v.Round, v.SetID, v.Message)
```